apps/parsley/src/components/LogRow/SubsectionHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
+import { palette } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { useLogWindowAnalytics } from "analytics";
+import Icon from "components/Icon";
+import { Row } from "components/LogRow/types";
+import { size, transitionDuration } from "constants/tokens";
+
+const { gray } = palette;
+
+interface SectionHeaderProps extends Row {
+  commandName: string;
+  functionID: string;
+  commandID: string;
+  onOpen: (v: {
+    commandID: string;
+    functionID: string;
+    isOpen: boolean;
+  }) => void;
+  open: boolean;
+}
+
+const SubsectionHeader: React.FC<SectionHeaderProps> = ({
+  commandID,
+  commandName,
+  functionID,
+  onOpen,
+  open,
+}) => {
+  const { sendEvent } = useLogWindowAnalytics();
+
+  return (
+    <Wrapper aria-expanded={open} data-cy="section-header">
+      <IconButton
+        aria-label="Click to open or close section"
+        data-cy="section-header-caret"
```

Comment:
Should this have a different attribute then SectionHeader?
```suggestion
        data-cy="subsection-header-caret"
```